prow-jobs/pingcap/ticdc/latest-presubmits.yaml

```diff
@@ -38,12 +38,24 @@ presubmits:
       always_run: false
       skip_if_only_changed: *skip_if_only_changed
       context: pull-cdc-kafka-integration-light
-      optional: true # TODO: change this after the job is ready
+      optional: false
       skip_report: false
       trigger: "(?m)^/test (?:.*? )?(pull-cdc-kafka-integration-light)(?: .*?)?$"
       rerun_command: "/test pull-cdc-kafka-integration-light"
       branches: *branches
 
+    - name: pingcap/ticdc/pull_cdc_kafka_integration_heavy
```

Comment:
Please add the following files based on pull_cdc_kafka_integration_light testing.
1. job DSL file https://github.com/PingCAP-QE/ci/blob/main/jobs/pingcap/ticdc/latest/pull_cdc_kafka_integration_light.groovy
2. pod file https://github.com/PingCAP-QE/ci/blob/main/pipelines/pingcap/ticdc/latest/pod-pull_cdc_kafka_integration_light.yaml
3. pipeline file https://github.com/PingCAP-QE/ci/blob/main/pipelines/pingcap/ticdc/latest/pull_cdc_kafka_integration_light.groovy